qa/repository-multi-version/src/test/java/org/elasticsearch/upgrades/MultiVersionRepositoryAccessIT.java

```diff
@@ -193,13 +194,25 @@ public void testReadOnlyRepo() throws IOException {
         }
     }
 
+    private static final List<Class<? extends Exception>> EXPECTED_BWC_EXCEPTIONS =
+            List.of(ResponseException.class, ElasticsearchStatusException.class);
+
     public void testUpgradeMovesRepoToNewMetaVersion() throws IOException {
         final String repoName = getTestName();
         try (RestHighLevelClient client = new RestHighLevelClient(RestClient.builder(adminClient().getNodes().toArray(new Node[0])))) {
             final int shards = 3;
             final String index=  "test-index";
             createIndex(client, index, shards);
-            createRepository(client, repoName, false);
+            final Version minNodeVersion = minimumNodeVersion();
+            // 7.11.1+ and newer will try to load RepositoryData during repo creation if verify is true, which is impossible in case
```

Comment:
```suggestion
            // 7.12.0+ and newer will try to load RepositoryData during repo creation if verify is true, which is impossible in case
```

We didn't backport to 7.11?

Reply:
Oh actually we did, for whatever reason my head decided we already released 7.11 now ..., strange that this wasn't failing in 7.11 though. I'll have to double check that. On it.